proto/config/appconfig.proto

```diff
@@ -235,3 +238,11 @@ message VolumeRef {
   // if mount_dir is empty then it will be mounted on /mnt
   string mount_dir = 3;
 }
+
+// AppRuntimeType - to indicate the types of Application Deployment
+// which may need special handling on the controller or on the device
```

Comment:
Presumably the controller can do special handling without it being in the EVE API, so drop "controller or" from the comment?

Can we document what IOT_EDGE implies (connect to port X for metrics and logs ...) and likewise for DOCKER?

Also, today the device handles Azure IoT Edge without any setting. How will we phase in setting 1 in the controllers and then having EVE-OS look for that value?

Reply:
Ok, reword the comments
I removed the IOT_EDGE item, so this is only for docker compose and future extension if needed for special handling.
We'll document this on the eve doc side when submitting EVE side of the changes related to this API